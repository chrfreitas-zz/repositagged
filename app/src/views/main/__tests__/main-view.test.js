import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Main from '../index';

describe('Main view', () => {
  it('should match with snapshot', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Main />);
    expect(result).toMatchSnapshot();
  });
});
