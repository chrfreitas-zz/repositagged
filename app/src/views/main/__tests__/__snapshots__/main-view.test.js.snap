// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main view should match with snapshot 1`] = `
<div
  className="main"
>
  <div
    className="main__header"
  >
    <Header
      title="githubstars"
    />
  </div>
  <div
    className="main__content"
  >
    Seja bem vindo!
  </div>
</div>
`;
